core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -50,9 +50,8 @@ <h4 class="bold-title">
                     <section class="service-box-section align-self">OR</section>
                     <section class="pad-left0 service-box-section">
                         <div tabindex="0" class="service-box" [ngClass]="{'service-box-selected': typeOfService == 'sls-app'}" (keyup.enter)="changeServiceType('sls-app')" (click)="changeServiceType('sls-app')">
-                            <div class="icon-icon-sls-app" [ngClass]="{'selected': typeOfService == 'sls-app'}">
-                                <img *ngIf="typeOfService == 'sls-app'" src="./../../../assets/images/icons/Build-custom.svg"/>
-                                <img *ngIf="typeOfService != 'sls-app'" src="./../../../assets/images/icons/Build-custom-gray.svg"/>
+                            <div class="icon-icon-sls-app selected">
+                                <img src="./../../../assets/images/icons/Build-custom.svg"/>
```

Comment:
How are these `img` blocks handled? I see them for this & `function`, but they don't exist for `api` & `website` types. Can we make this consistent?

Reply:
Discussed with @pgobin, we'll keep this for now since we are not introducing this change now.